server/@types/viewModels/index.d.ts

```diff
@@ -9,7 +9,7 @@ declare module 'viewModels' {
   }
 
   export interface PrisonerSupportNeeds {
-    problemRetrievingData?: boolean
```

Comment:
These `problemRetrievingData` fields don't need to be optional - there are no cases where we don't set them (nor indeed should there be I don't think)